src/stories/molecules/Tabs.tsx

```diff
@@ -0,0 +1,70 @@
+import React, { Children, useState } from 'react';
+import './tabs.css';
+
+export interface TabsProps {
+  children?: React.ReactNode;
+}
+
+type TabTitleProps = { children: React.ReactNode };
+type TabContentProps = { children: React.ReactNode };
+type TabProps = { id: string, children: React.ReactNode };
+
+const TabsFC = ({children}: TabsProps) => {
+  const tabArray = (Children.toArray(children || []).filter(
+    (child : React.ReactElement) => child.type === Tabs.Tab)) as React.ReactElement<TabProps>[];
+  const [activeTab, setActiveTab] = useState(0);
+
+  function c_id(child: React.ReactElement<TabProps>) {
+    return child.props.id;
+  }
+
+  function c_title(children: React.ReactNode) {
+    const title = (Children.toArray(children || []).find(
+      (child : React.ReactElement) => child.type === Tabs.Tab.Title)) as React.ReactElement<TabTitleProps>;
```

Comment:
The type after `as` should have `[]` at the end, just like on line 14.

Reply:
Should we have `[] `even if I used `.find` that return only one element? In case there are more than one title, I get the first one.